code/__HELPERS/game.dm

```diff
@@ -535,4 +535,14 @@
 	return hex2num(copytext(hexa, 4, 6))
 
 /proc/GetBluePart(const/hexa)
-	return hex2num(copytext(hexa, 6, 8))
\ No newline at end of file
+	return hex2num(copytext(hexa, 6, 8))
+
+/proc/get_opposite_dir(direction) //returns the direction opposite the input direction.

```

Comment:
why do people always define a proc for this?

```turn(dir, 180)``` works in all cases (I don't know about UP, DOWN, but I suspect it probably does, not to mention we don't use those (yet))